test/controllers/beneficial.owner.individual.controller.spec.ts

```diff
@@ -856,7 +856,7 @@ describe("BENEFICIAL OWNER INDIVIDUAL controller", () => {
   });
 
   describe("POST with url params tests", () => {
-    test(`redirects to ${BENEFICIAL_OWNER_TYPE_PAGE} page`, async () => {
+    test.only(`redirects to ${BENEFICIAL_OWNER_TYPE_PAGE} page`, async () => {
```

Comment:
.only needs to be removed

Reply:
This file was committed in error. Have reverted change.